rudder-web/pom.xml

```diff
@@ -55,11 +55,20 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
   </repositories>
 
   <build>
+    <resources>
```

Comment:
After this ressources is added i don't have any other ressources in my build (/target/classes) I don't know if this is ok when the webapp is built or not.

If we keep it we will need to provide much more information in the jetty config on devellepment environement